### PR TITLE
Update Vocabulary, goals and requirements

### LINKED ISSUE #1: Clarification of terminology and scope
The README document refers to both `Web content` and `content accessible on the Web`, but it is not clear to me if (a) they are the same thing, in this context and (b) what exactly is meant.   There are also references in the README to various asset formats - images, videos, text, etc.  Does this CG plan to address all of these types of assets?  And if so, in what publishing or distribution contexts?

### PR DESCRIPTION
**Pull request by llemeurfr**
This PR fixes #1, #3, #4, #5, #6.

It contains an update to the Vocabulary and and evolution of the Requirements (renamed Goals and Requirements).

### CODE REVIEW

File: docs/requirements.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-## High level requirements: 
+## Goals
 
-To ensure that all users without discrimination will be able to reserve their rights, the solution shall:
+To ensure that all users without discrimination will be able to reserve TDM Rights, the solution shall:
 
-1. Rely on open standards and be interoperable.
-1. Rely on tools and formats widely supported by web crawlers.
-1. Be easy to implement, with no economic or technical barriers.
+1. Rely on open standards  - especially Web standards - and be interoperable.
```
Comment (lrosenthol) [inline docs/requirements.md:7]: As there is no formal definition for what constitutes a "Web standard",  I would strongly *object* to this addition.

File: docs/requirements.md [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
+1. Rely on tools and formats widely supported by publishers and TDM Actors.
```
Comment (lrosenthol) [inline docs/requirements.md:8]: We don't have a definition for _publisher_ - so either we need to define it *or* not use it.

File: docs/requirements.md [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
+1. Specify how a rightsholder can indicate the location of a TDM Licence associated with each individual Web resource he controls.
+1. Specify a machine-readable format for TDM Licenses. 
 
-Although this should not over-complexify the solution, it should still:
+## Additional Requirements for Advanced Solutions
 
-1. Allow the rightsholder to specify whether i) all TDM rights (other than for scientific research purposes) are reserved or ii) only TDM rights for commercial purposes are reserved  
-1. Specify how a rightsholder can indicate that TDM rights are reserved on random Web resources he controls (e.g. images and video files which may be found by crawlers outside of Web pages).
\ No newline at end of file
+Advanced user needs or sector specific practices may imply additional requirements. The additional features resulting from these additional requirements will not replace the code features resulting from core requirements, but will complement them.
```
Comment (lrosenthol) [inline docs/requirements.md:24]: Remove the word `code` (ie. "code features").

File: docs/requirements.md [hunk 4]
```diff
@@ -1,26 +1,26 @@
...
+The solution shall:
 
-## Secondary requirements
+1. Specify how a rightsholder can declare the reservation of TDM Rights on each individual Web resource he controls.
+1. Specify how a rightsholder can indicate the location of human readable information indicating how to apply for a TDM License.
+1. Specify how a rightsholder can indicate the location of a TDM Licence associated with each individual Web resource he controls.
+1. Specify a machine-readable format for TDM Licenses. 
 
-Although this should not over-complexify the solution, it should still:
+## Additional Requirements for Advanced Solutions
```
Comment (lrosenthol) [inline docs/requirements.md:22]: I don't like the term "Advanced Solutions", can we find a better term - or just leave it as a "Additional Requirements"?

**Review by lrosenthol — CHANGES_REQUESTED** [review]
Some good stuff, but a few changes that I think aren't fully worked through yet...

File: docs/requirements.md (hunk 1 above)
Comment (llemeurfr) [reply docs/requirements.md:7]: That's not an important point, let's remove.

File: docs/requirements.md (hunk 2 above)
Comment (llemeurfr) [reply docs/requirements.md:8]: It's difficult to replace it by rightsholder, as rightsholders often rely on other parties to publish their content on the web.
So ok let's define it (see update of the Vocabulary).

File: docs/requirements.md (hunk 3 above)
Comment (llemeurfr) [reply docs/requirements.md:24]: Typo; we were meaning "core".

File: docs/requirements.md (hunk 4 above)
Comment (llemeurfr) [reply docs/requirements.md:22]: As the context is explained in the body, we can remove "for Advanced Solutions" from the title.

**Comment by lrosenthol** [comment]
Thanks for the changes!

File: docs/requirements.md [hunk 5]
```diff
@@ -1,26 +1,26 @@
...
 
-## Primary requirements
+## Core requirements
 
-1. Specify how a rightsholder can declare TDM rights on a Web page he controls.
-1. Specify how a rightsholder can declare TDM rights on each individual Web resource he controls, when such Web resources are present in a Web page.
-1. Specify how the rightsholder can indicate whether a licence for TDM rights is available (in case TDM rights are reserved).
-1. Specify how a TDM actor can apply for a TDM license.
-1. Specify a machine-readable format for TDM licenses. 
+The solution shall:
```
Comment (bquinn) [inline docs/requirements.md:15]: As I suggested in the meeting, we should change "The solution shall" to "The technical specification shall", to make it clear that it's the spec that will include all of these things, and implementations don't necessarily have to cover all of these requirements.

Comment (llemeurfr) [reply docs/requirements.md]: The requirements file has been updated accordingly.

**Review by bquinn — APPROVED** [review]

**Pull request merged**